plutus/src/main/java/com/bloxbean/cardano/client/plutus/spec/serializers/PlutusDataJsonConverter.java

```diff
@@ -63,4 +69,94 @@ else if (jsonNode.has(LIST)) {
     public static String toJson(PlutusData plutusData) throws JsonProcessingException {
         return mapper.writeValueAsString(plutusData);
     }
-}
+
+    /**
+     * Convert {@link PlutusData} to utf8 encoded json String
+     * @param plutusData data to be encoded
+     * @return utf8 encoded json
+     * @throws JsonProcessingException
+     * @throws CborSerializationException
+     */
+    public static String toUTF8Json(PlutusData plutusData) throws JsonProcessingException, CborSerializationException {
```

Comment:
This is specific to metadata used in PlutusData and may not be valid for regular Plutus data. I think it would be better to move this to another helper class or rename this method.

Can you move this method to another PR? I think we can then decide on the name, right package for this helper and add some more tests in that PR.